spec/features/staff/users_can_choose_recipient_country_spec.rb

```diff
@@ -55,28 +55,11 @@
         expect(page).to have_selector "li.autocomplete__option", text: "Afghanistan", visible: true
       end
 
-      scenario "typing a complete country name, clicking it in the list and clicking continue saves the country" do
-        fill_in I18n.t("form.label.activity.recipient_country"), with: "Saint Lucia"
-        find("li.autocomplete__option", text: "Saint Lucia").click
-        click_button I18n.t("form.button.activity.submit")
-        click_on I18n.t("default.link.back")
```

Comment:
I presume this more specific feature test exists to give greater coverage to the country selection journey which uses a combo of JS and no-JS to provide a different experience.

Instead of removing the test would it be worth changing the way the assertion is made? Perhaps it could navigate through the new "Activities" in the nav, select the new activity and check the content as the user would now be expected to do.